lib/cmsis_rtos_v1/cmsis_mailq.c

```diff
@@ -167,6 +167,11 @@ osStatus osMailFree(osMailQId queue_id, void *mail)
 {
 	osMailQDef_t *queue_def = (osMailQDef_t *)queue_id;
 
+	/* Note: Below 2 error codes are not supported.
+	 * osErrorValue: mail block does not belong to the mail queue pool.
+	 * osErrorParameter: the value to the parameter queue_id is incorrect.
+	 */
+
```

Comment:
Remove these comments. They will be put in the doc section separately.

Reply:
I saw similar comments mentioned in Mutex, Semaphore and Mempool.
These lines were taken from lib/cmsis_rtos_v1/cmsis_mempool.c
Shall i remove it even in those places ?

Reply:
Got it. Addressed the comment.